src/main/scala/CSR.scala

```diff
@@ -4,12 +4,27 @@ import chisel3.util._
 class CSR extends Module {
   val io = IO(new CSRBundle)
 
-  val mie = RegInit(0xfff.U(12.W))
+  // to do: fill the blanks
+  // val mvendorid = 
+  // val marchid = 
+  // val mimpid = 
+  // val mhardid = 
+
   val mstatus = RegInit(1.U(32.W) << MIRField.MIE)
+  val misa = "h40000000".U //
```

Comment:
> The Extensions field encodes the presence of the standard extensions, with a single bit per letter of the alphabet (bit 0 encodes presence of extension “A” , bit 1 encodes presence of extension “B”, through to bit 25 which encodes “Z”). The "I" bit will be set for RV32I, RV64I, RV128I base ISAs, and the “E” bit will be set for RV32E.

Please set the "I" bit, because we already support most (and plan to support all) of the "I" instruction set.
And please add reading code in the `when` block below.